cd/deploy.sh

```diff
@@ -1,5 +1,75 @@
 #!/bin/bash
 
-# Release to Dockerhub (if the branch is master)
+set -e
 
-# Update develop
\ No newline at end of file
+# Gets project root directory by calling two nested "dirname" commands on the this file 
+getRootDirectory() {
+    echo "$( cd "$(dirname "$(dirname "${BASH_SOURCE[0]}")")" > /dev/null 2>&1 && pwd )"
```

Comment:
I don't think this gets the root directory. It seems to get the directory that the script is being run from:

```
gaffer-docker$ ./cd/deploy.sh 
/home/ec2-user/gaffer-docker

cd$ ./deploy.sh 
/home/ec2-user/gaffer-docker/cd
```